src/Repository/ConferenceRepository.php

```diff
@@ -85,6 +85,17 @@ public function getAllConferencesAsRow(): array
             ;
     }
 
+    /** @return array<Conference> */
+    public function getNullCoordinatesConferences(): array
+    {
+        return $this->createQueryBuilder('c')
+            ->andWhere('c.online = false')
```

Comment:
We should use a parameter here, since all RDBMS does not use the same values for boolean (i.e: pgsql accepts `False`, `false`, `0` at least)